static/src/javascripts/projects/commercial/modules/dfp/on-slot-render.js

```diff
@@ -75,14 +75,18 @@ export const onSlotRender = (event: SlotRenderEndedEvent): void => {
         // Set refresh field based on the outcome of the slot render.
         const sizeString = advert.size && advert.size.toString();
         const isNotFluid = sizeString !== '0,0';
+        const isOutstream = sizeString === '620,350';
```

Comment:
Anything more solid than that? Does it work at all breakpoints?

Reply:
At the moment it works at desktop.  There will be some more identifying sizes for the mobile breakpoint, yet to be determined.